.editorconfig

```diff
@@ -13,6 +13,10 @@ tab_width = 4
 
 #### .NET Coding Conventions ####
 
+# Organize using directives
+dotnet_sort_system_directives_first = true
```

Comment:
added this based upon feedback from Dag, we can re format the files again to accomodate this rule

Reply:
the pr is already ~800 lines

Reply:
at ~800, we've already broken the rule

Reply:
Btw, it says here **nongenerated** code change:

<img width="756" alt="Screenshot 2025-03-11 at 15 56 38" src="https://github.com/user-attachments/assets/df3beb40-658a-4a15-ab84-92d0f27d4f17" />

https://cognitedata.slack.com/archives/C08DKEDUN05/p1741260232912359

Reply:
ah, oki true